src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -5,6 +5,7 @@ import {
   Animated,
   StatusBar,
   StyleSheet,
+  Text,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Text` component is added which is used later in the code. This change seems fine.
```diff
+  Text,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->